.pre-commit-config.yaml

```diff
@@ -29,3 +29,15 @@ repos:
       - id: pydocstyle
         additional_dependencies: [toml]
         args: ["--ignore=D202,D203,D205,D401,D212"]
+
+  - repo: local
```

Comment:
Why did you configure the local pylint installed with poetry?

Reply:
because it said in the pylint's pre-commit hook documentation that it needs to be installed locally to have access to local modules and packages. 

> Since pylint needs to import modules and dependencies to work correctly, the hook only works with a local installation of pylint (in your environment). 

from: https://pylint.pycqa.org/en/latest/user_guide/installation/pre-commit-integration.html